### ISSUE TITLE
[BUG]  413 HTTP response returned when uploading large files via LiveResponse

### ISSUE THREAD

**Issue by 7a6570** [carbonblack/carbon-black-cloud-sdk-python#543]
**I am seeing this behaviour on:**
 - OS: Linux
 - Carbon Black Cloud Products: Enterprise EDR, Audit & Remediation
 - Python Version: 3.11.2

**Describe the bug**
When uploading large files to endpoints via LiveResponse, a HTTP 413 response is returned from the backend:

```
<html>\r\n<head><title>413 Request Entity Too Large</title></head>\r\n<body>\r\n<center><h1>413 Request Entity Too Large</h1></center>\r\n<hr><center>nginx</center>\r\n</body>\r\n</html>
```

**Steps to Reproduce**

Upload a large file (e.g. 40MB) to an endpoint using the cbc sdk:

```
with endpoint.lr_session() as lr_session:
  lr_session.put_file(fp, dest_path)
```

**Expected behavior**
Until recently, it was possible to upload a 40MB file to clients using the cbc SDK.

**Comment by 7a6570** [comment]
I did some additional testing: if the file is smaller than 1MB, the upload still works, otherwise it fails returning the HTTP 413 header. 
Uploading the large file (40MB) via browser seems to work though.
Can you please into this issue ? thx

**Comment by avanbrunt-cb** [comment]
What version of the SDK are you using as we attempted a fix for this in 1.5.6 with this PR https://github.com/carbonblack/carbon-black-cloud-sdk-python/pull/533

**Comment by 7a6570** [comment]
> Sorry for the slow response. Your saying this was support previously but has stopped working? Do you know when that stopped working?

Yes this worked for a long time and stopped working maybe three or four weeks ago.

> What version of the SDK are you using as we attempted a fix for this in 1.5.6 with this PR 

I'm using version 1.5.6

**Comment by avanbrunt-cb** [comment]
@7a6570 Can you open a support case with us and drop the case number here. Nothing has changed in the SDK so this is likely the backend API somewhere

**Comment by avanbrunt-cb** [comment]
We made one minor change in 1.5.6 with this PR https://github.com/carbonblack/carbon-black-cloud-sdk-python/pull/533

Can you try using 1.5.5 and letting me know if that makes a difference

**Comment by avanbrunt-cb** [comment]
We have confirmed a backend issue and it is being worked on

**Comment by 7a6570** [comment]
> We have confirmed a backend issue and it is being worked on

OK, thanks for the update. I already opened a case, the number is: 35912150
